README.md

```diff
@@ -93,6 +93,12 @@ If you have any questions about Cortex:
 
 Your feedback is always welcome.
 
+## Community Meetings
+
+The Cortex community call happens every three weeks on Thursday at 04:30pm – 05:15pm GMT+1 to get calender invite join the [google groups](https://groups.google.com/forum/#!forum/cortex-monitoring) 
```

Comment:
I would suggest to use UTC, so `03:30pm – 04:15pm UTC`, and please add a `.` at the end of the sentence.